src/test/java/org/jenkinsci/plugins/scriptsecurity/scripts/ClasspathEntryTest.java

```diff
@@ -39,9 +46,23 @@
         }
         assertEquals("jar:file:/tmp/x.jar!/subjar.jar", "jar:file:/tmp/x.jar!/subjar.jar");
```

Comment:
Seems I screwed up in #2; this is supposed to be `assertRoundTrip`. Could you fix it please?